pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -1189,3 +1189,24 @@ def label(self, decimals=None):
 
     def adjoint(self):
         return Barrier(wires=self.wires)
+
+
+class WireCut(Operation):
+    r"""WireCut(wires)
+    The wire cut operator, used to manually place qubit cut locations.
```

Comment:
```suggestion
    The wire cut operator, used to manually mark locations for wire cuts.
```
Might not matter too much, but wanted to emphasize that this operation is more of a marker than actually placing the cut.